### PR TITLE
[#229] Add Github Actions Templates document

### LINKED ISSUE #229: Add Github Actions Templates document
## Why

As we have [a Github Action workflow](https://github.com/nimblehq/ios-templates/pull/224) that supports publishing docs to the wiki, we need to bring all iOS templates' documents into the repository's source.

## Acceptance Criteria

- Add document into the folder `.github/wiki`

- Brings document from [the Notion page](https://www.notion.so/nimblehq/Github-Actions-d009345c7ffb40768fb81d316a46984f) into the repository's source:
  - [ ] Github Actions Templates

### PR DESCRIPTION
**Pull request by blyscuit**
close #229 

## What happened

Add document into the folder .github/wiki

Brings document from [the Notion page](https://www.notion.so/nimblehq/Github-Actions-d009345c7ffb40768fb81d316a46984f) into the repository's source:

 
## Proof Of Work
Screen Shot 2022-05-19 at 10 25 45

### CODE REVIEW

File: .github/wiki/Github-Actions.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Github Actions
+
+Use GitHub Actions template to start a new project with GitHub Actions as the CICD tool.
```
Comment (suho) [inline .github/wiki/Github-Actions.md:3]:
```suggestion
Use the GitHub Actions template to start a new project with GitHub Actions as the CI/CD tool.
```

File: .github/wiki/Github-Actions.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+|SSH_PRIVATE_KEY         |SSH key connected to a user with access to the match repo for check out the match repo.                                             |-   |✅              |✅                      |✅                                  |
+|MATCH_PASS              |Fastlane Match Passphrase for decrypting a match repository.                                                                        |-   |✅              |✅                      |✅                                  |
+|APPSTORE_CONNECT_API_KEY|App Store Connect API https://docs.fastlane.tools/actions/app_store_connect_api_key/ for uploading build to TestFlight or App Store.|-   |-              |-                      |✅                                  |
+|FIREBASE_TOKEN          |Firebase token https://firebase.google.com/docs/cli#cli-ci-systems for uploading build to Firebase Distributions and Analytics.     |-   |✅              |✅                      |✅ For uploading dSYM to Crashlytics|
+
+## Installation
+
+1. Following the setup instruction in `Readme.md`.
```
Comment (suho) [inline .github/wiki/Github-Actions.md:60]:
```suggestion
1. Following the setup instruction in `README.md`.
```

**Review by suho — APPROVED** [review]

**Comment by suho** [comment]
@blyscuit One more thing, please help me to add this document to the `_Sidebar.md` file.

File: .github/wiki/Github-Actions.md (hunk 1 above)
Comment (suho) [reply .github/wiki/Github-Actions.md:3]: @blyscuit I think you missed this one :D

**Review by markgravity — APPROVED** [review]
LGTM 👍

**Pull request merged**